content/shared/influxdb3-query-guides/snapshots/snowflake.md

```diff
@@ -1,185 +1,143 @@
+- Setting up a Snowflake external stage using the `CREATE STAGE` Snowflake SQL command and your storage provider details.
+This establishes an external storage location (such as AWS S3) to store Iceberg table data and metadata.
+- Setting up a catalog integration in Snowflake to manage and load Iceberg tables efficiently.
+For more information, refer to the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/tables-iceberg-configure-catalog-integration).
+- Creating an export configuration that tells the InfluxDB Iceberg exporter which tables are available for export.
 
-#### Example: Create a catalog integration in Snowflake
+After the setup is complete, you can export data snapshots to Iceberg format and query the Iceberg table from Snowflake.
 
-```sql
-CREATE CATALOG INTEGRATION my_catalog_integration
-  CATALOG_SOURCE = 'OBJECT_STORE'
-  TABLE_FORMAT = 'ICEBERG'
-  ENABLED = TRUE;
-```
+> [!Important]
+> #### Export tables are read-only in Snowflake
+> 
+> Tables created through this integration are _read-only_.  
+> You cannot write directly to these tables using Snowflake or any other engine. 
+> 
 
-For more information, refer to the [Snowflake documentation](https://docs.snowflake.com/en/user-guide/tables-iceberg-configure-catalog-integration).
+## Export data to Iceberg format
 
-### Export InfluxDB time series data to Iceberg format
+Use the `influxctl` CLI command or the HTTP API to export snapshots to Iceberg format.
 
-> **Note**: Before exporting InfluxDB time series data to Iceberg format, ensure that the relevant InfluxDB tables are properly set up. Please reach out to your support engineers to configure the tables that need to be exported.
+> [!Important]
+> Before you can export data to Iceberg format, the InfluxData Support Team needs
+> to have set up your tables for exporting.
+> For more information, [contact the InfluxData Sales team]({{< cta-link >}}).
 
 {{< tabs-wrapper >}}
 {{% tabs %}}
-[CLI](#tab-cli)
-[API](#tab-api)
+[CLI](#cli)
+[API](#api)
 {{% /tabs %}}
-
 {{% tab-content %}}
-{{% tab-pane id="tab-cli" %}}
-#### Using the CLI
 
-Use the `influxctl` command to export InfluxDB time-series data to Iceberg format:
-
-```sh
-influxctl snapshot export --namespace foo --table bar
+{{% code-placeholders "NAMESPACE|TABLE_NAME" %}}
+```bash
+influxctl snapshot export --namespace NAMESPACE --table TABLE_NAME 
 ```
-{{% /tab-pane %}}
-{{% tab-pane id="tab-api" %}}
-#### Using the API
-
-Use the {{% product-name %}} HTTP API to export snapshots and check status.
+{{% /code-placeholders %}}
 
-##### Example: Export a snapshot
+Replace the following:
 
-This example demonstrates how to export a snapshot of your data from InfluxDB to an Iceberg table using the HTTP API.
+- {{% code-placeholder-key %}}`NAMESPACE`{{% /code-placeholder-key %}}: The namespace of the data to export. <!-- Namespace might need more explanation --> 
```

Comment:
I think the namespace in Dedicated is just the database name, but we'll have to confirm.

Reply:
> I think the namespace in Dedicated is just the database name, but we'll have to confirm.

I think so, too - had to look it up, but I'm not 100% sure.